libs/barista-components/sunburst/src/sunburst.util.ts

```diff
@@ -0,0 +1,466 @@
+/**
+ * @license
+ * Copyright 2020 Dynatrace LLC
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {
+  DtColors,
+  DT_CHART_COLOR_PALETTE_ORDERED,
+} from '@dynatrace/barista-components/theming';
+import * as d3 from 'd3-shape';
+
+export interface DtSunburstNode {
+  label: string;
+  value?: number;
+  color?: DtColors | string;
+  children?: DtSunburstNode[];
+}
+
+export enum DtSunburstValueMode {
+  ABSOLUTE = 'absolute',
+  PERCENT = 'percent',
+}
```

Comment:
I think we have discussed in #786 to use a type for this, as this will fit the guidelines of the library better. 
If we did not finish that conversation there, than I am sorry about that. 

We try to stay away from enums in the library as they bloat the resulting bundle quite significantly when not made a constant.